hoomd/md/TwoStepRATTLELangevinGPU.cu

```diff
@@ -206,12 +206,21 @@ gpu_rattle_langevin_angular_step_two(const Scalar4* d_pos,
     dim3 grid((group_size / block_size) + 1, 1, 1);
     dim3 threads(block_size, 1, 1);
 
+    unsigned int shared_bytes
+        = max((unsigned int)(sizeof(Scalar3) * rattle_langevin_args.n_types),
+              (unsigned int)(rattle_langevin_args.block_size * sizeof(Scalar)));
```

Comment:
```suggestion
   const auto shared_bytes
        = max(static_cast<size_t>(sizeof(Scalar3) * rattle_langevin_args.n_types),
              static_cast<size_t>(rattle_langevin_args.block_size * sizeof(Scalar)));
```

Reply:
Static casting to a size_t is not necessary as `sizeof()` returns a `size_t`. I don't know why I kept all these `unsigned ints` around as CUDA accepts a `size_t` for the shared memory bytes: https://docs.nvidia.com/cuda/cuda-c-programming-guide/index.html#execution-configuration

I will adjust accordingly.